.github/workflows/release.yml

```diff
@@ -128,27 +126,22 @@ jobs:
       - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
         with:
           go-version: "^1.18"
+      - uses: mlugg/setup-zig@v2
+        with:
+          version: 0.14.1
       - name: Install cargo-dist
```

Comment:
We tend not to use git tags to lock actions, we need to use the SHA instead.

We should also make sure to review `setup-zig` since it doesn't appear to be an official product from the zig project.